### ISSUE TITLE
warning: rule "ransomware_sodinokibi" in .../RANSOM_Sodinokibi.yar(28): expression always false - requesting 8 of 4.

### ISSUE THREAD

**Issue by gitezri** [advanced-threat-research/Yara-Rules#20]
warning: rule "ransomware_sodinokibi" in .../RANSOM_Sodinokibi.yar(28): expression always false - requesting 8 of 4.

**Comment by ThisIsLibra** [comment]
Good catch, fixed it in [1919562](https://github.com/advanced-threat-research/Yara-Rules/commit/1919562a59f190bda60c982424f6a24c542ee3e0)!
